### PR TITLE
add map_fps fig

### PR DESCRIPTION
**Pull request by heavengate**
**add map_fps figure**

### CODE REVIEW

File: deploy/python/infer.py [hunk 1]
```diff
@@ -517,7 +517,7 @@ def predict_video():
     fps = 30
     width = int(capture.get(cv2.CAP_PROP_FRAME_WIDTH))
     height = int(capture.get(cv2.CAP_PROP_FRAME_HEIGHT))
-    fourcc = cv2.VideoWriter_fourcc(* 'mp4v')
```
Comment (jerrywgz) [inline deploy/python/infer.py:520]: is this change necessary?

Comment (heavengate) [reply deploy/python/infer.py:520]: `pre-commit` format changes..

Comment (still-wait) [reply deploy/python/infer.py:520]: `pre-commit` specification
